.travis.yml

```diff
@@ -12,6 +11,9 @@ php:
 matrix:
     include:
         - php: 5.3
+          dist: precise
+        - php: 5.3
```

Comment:
Use a newer PHP version for this prefer-lowest job, so that we don't need to have multiple jobs on the precise environment (which does not have persistent cache support for OSS as they dropped the sudo-disabled one for precise).
I suggest putting this job on 5.6